core/controllers/questions_list.py

```diff
@@ -47,7 +47,11 @@ class QuestionsListHandler(base.BaseHandler):
     @acl_decorators.open_access
     def get(self, comma_separated_skill_ids):
         """Handles GET requests."""
-        start_cursor = self.request.get('cursor')
+        offset = self.request.get('cursor')
+        if offset == '':
+            offset = 0
+        else:
+            offset = int(offset)
```

Comment:
How hard would it be to modify the frontend to send numbers and not cursor?

Reply:
Modified the frontend. Done.